contrib/zerotier-systemd-manager.timer

```diff
@@ -1,12 +1,9 @@
 [Unit]
 Description=Update zerotier per-interface DNS settings
-Requires=zerotier-systemd-manager.service
 
 [Timer]
-Unit=zerotier-systemd-manager.service
-OnStartupSec=60
-OnUnitInactiveSec=60
-Persistent=true
+OnStartupSec=2min
```

Comment:
any specific reason this doubled?

Reply:
Startup of a PC can take a long time and pushing this a little further avoids too much polluting of the journal until network starts properly.
I noticed on my PC that even though the .service is ordered After=zerotier-one.service, the .timer still fires up at the scheduled time, just not starting anything.

Reply:
So this is all very relative and arbitrary, but my pc boots in <10 seconds to a functioning desktop and a lot of others do too.

I don't want to preclude making sure you get the desired result, but you may see how this is an issue for me, who will need to wait for 2+ minutes for the timer to start, unless I am misunderstanding something.

Is there a better solution here that you know of?

Reply:
better == something that resolves both of our timing issues.

Reply:
I can't test right now, but you should be able to use 
```
Requires=zerotier-one.service
After=zerotier-one.service
```
In the _.timer_

Reply:
I just realized that this is a non-issue: the behavior I described occurs only at the boot of the unconfigured system, and the journal is "polluted" nonetheless by the normal execution of the timer every minute.

@laduke I specifically mentioned no dependencies in the .timer unit in my "best practices"

Therefore, I reverted my change on this parameter with my latest commit in my repo.

Reply:
Can you point me towards the best practices info? I don't see that in the arch wiki.

Reply:
thanks @laduke and @devvick for sorting through this. I appreciate it.

I'll review in a little bit and try to merge in the right thing after some testing.

Reply:
> Can you point me towards the best practices info? I don't see that in the arch wiki.
@laduke
Sorry, English is my second language and I probably should have used different wording. What I meant literally was: "my-arbitrarily-worked-out-through-experience-and-docs-reading **good** practices of configuring systemd".

The short version of my reasoning behind the first point is: Timers are not services, and they intrinsically depend on the calendar itself or their monotonic schedule to launch them.